doc/api/fs.md

```diff
@@ -1549,6 +1549,23 @@ Synchronous version of [`fs.readFile`][]. Returns the contents of the `file`.
 If the `encoding` option is specified then this function returns a
 string. Otherwise it returns a buffer.
 
+_Note: The behavior of `fs.readFileSync()` is platform specific. As such,
```

Comment:
@jasnell If i were to do this, this would be first line fs.readFile().
Would we need the same to be copied even for fs.readFileSync() ? 
Also , would I be giving examples for both sync and async counterparts ? 

@